packages/crayons-extended/custom-objects/src/components/form-builder/components/fb-field-dropdown.tsx

```diff
@@ -104,7 +149,7 @@ export class FbFieldDropdown {
       });
 
       if (boolElementUpdated) {
-        this.dataProvider = [...arrChoices];
+        this.dataProvider = this.level ? [...clonedChoices] : [...arrChoices];
```

Comment:
can we add (this.isDependentField && this.level) here

Reply:
Done.